tools/run_tests/run_xds_tests.py

```diff
@@ -48,6 +49,13 @@
 logger.addHandler(console_handler)
 logger.setLevel(logging.WARNING)
 
+# Suppress excessive logs for gRPC Python
+original_grpc_trace = os.environ.pop('GRPC_TRACE')
+original_grpc_verbosity = os.environ.pop('GRPC_VERBOSITY')
+# Suppress not-essential logs for GCP clients
+logging.getLogger('google_auth_httplib2').setLevel(logging.WARNING)
+logging.getLogger('googleapiclient.discovery').setLevel(logging.WARNING)
```

Comment:
What logs will this keep? We still want _some_ client logs though.

Reply:
This eliminates the "Make Requests" and "URL being requested" stuff. If the underlying HTTP failed, it will raise an exception that we won't miss.

What GCP client log do you think we should preserve here?

Reply:
I'm not exactly sure what we should preserve. I was thinking we need to keep the http requests it sends, and something to indicate that it was successful.

Do you have the log after this change?

Reply:
Here is a one-test-case log which creates/destroys GCP resources: https://paste.googleplex.com/4985518489075712

It might take sometime to generate the full log.

Reply:
Here is the full simplified log: https://gist.github.com/lidizheng/591d294f3725d7076b3fecdea3350488